funnel/static/translations/hi_IN/messages.json

```diff
@@ -1678,16 +1685,16 @@
       "Switch camera": [null, "कैमरा बदलें"],
       "Check in status": [null, "स्थिति की जांच करें"],
       "Scanning": [null, "स्कैन हो रहा है"],
-      "Hello %s": [null, ""],
+      "Hello %s": [null, "नमस्ते %s"],
       "Thank you for choosing to participate in  %(title)s and for being a part of this growing community.": [
         null,
-        " %(title)s में भाग लेने के चुनाव के लिए और इस बढ़ते कम्यूनिटी का हिस्सा बनने के लिए धन्यवाद।"
+        " %(title)s में भाग लेने के चुनाव के लिए और इस बढ़ते कम्यूनिटी का हिस्सा बनने के लिए धन्यवाद."
```

Comment:
No periods. Also, where is this text coming from? Source is not as per style guide.

Reply:
This text is coming from `badge.js.jinja2`